akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectives.scala

```diff
@@ -210,10 +202,9 @@ trait FileAndResourceDirectives {
 }
 
 object FileAndResourceDirectives extends FileAndResourceDirectives {
-  private val withRangeSupportAndPrecompressedMediaTypeSupportAndExtractSettings =
+  private val withRangeSupportAndPrecompressedMediaTypeSupport =
     RangeDirectives.withRangeSupport &
-      CodingDirectives.withPrecompressedMediaTypeSupport &
-      BasicDirectives.extractSettings
```

Comment:
no need to extract settings as the settings were only used to get the file io dispatcher config